.flutter-plugins

```diff
@@ -7,4 +7,11 @@ firebase_core=C:\\Users\\131la\\Downloads\\flutter_windows_2.5.2-stable\\flutter
 firebase_core_web=C:\\Users\\131la\\Downloads\\flutter_windows_2.5.2-stable\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\firebase_core_web-1.1.0\\
 flutter_facebook_auth=C:\\Users\\131la\\Downloads\\flutter_windows_2.5.2-stable\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\flutter_facebook_auth-3.5.2\\
 flutter_facebook_auth_web=C:\\Users\\131la\\Downloads\\flutter_windows_2.5.2-stable\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\flutter_facebook_auth_web-2.6.0+2\\
+flutter_plugin_android_lifecycle=C:\\Users\\131la\\Downloads\\flutter_windows_2.5.2-stable\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\flutter_plugin_android_lifecycle-2.0.3\\
+geolocator=C:\\Users\\131la\\Downloads\\flutter_windows_2.5.2-stable\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\geolocator-7.7.0\\
+geolocator_android=C:\\Users\\131la\\Downloads\\flutter_windows_2.5.2-stable\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\geolocator_android-2.1.0\\
+geolocator_apple=C:\\Users\\131la\\Downloads\\flutter_windows_2.5.2-stable\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\geolocator_apple-1.2.2\\
+geolocator_web=C:\\Users\\131la\\Downloads\\flutter_windows_2.5.2-stable\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\geolocator_web-2.0.6\\
+google_maps_flutter=C:\\Users\\131la\\Downloads\\flutter_windows_2.5.2-stable\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\google_maps_flutter-2.0.11\\
+google_maps_flutter_web=C:\\Users\\131la\\Downloads\\flutter_windows_2.5.2-stable\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\google_maps_flutter_web-0.3.2\\
```

Comment:
In future please do not commit this, or other configuration files to the repository.

**TODO**
- Add .flutter-plugins to .gitignore